src/cmd/package.go

```diff
@@ -62,7 +62,20 @@ func newPackageCommand() *cobra.Command {
 	return cmd
 }
 
-type packageCreateOptions struct{}
+type packageCreateOptions struct {
```

Comment:
We want to move to this strategy overall, I am doing it in create for this PR so that I can write unit tests at the CLI level rather than e2e tests